docs/source/howtos/optimise_experiment.rst

```diff
@@ -0,0 +1,176 @@
+    general advice is given, but consult a domain-area expert for any science
+    concerns. Different experiments *will* require different considerations.
+
+
+Using resolution plots
+----------------------
+
+The optimal experimental :term:`settings<setting>` can be estimated using purely
+ResINS and a plotting library, though some knowledge of the system under
+investigation is still required. To do this, the :term:`instrument`, the
+:term:`model`, and the :term:`option` for each :term:`configuration` have to
+have been chosen. The first step is to create the instrument:
+
+>>> from resolution_functions import Instrument
+>>> maps = Instrument.from_default('MAPS')
+>>> print(maps)
+Instrument(name=MAPS, version=MAPS)
+
+and then find the :term:`settings<setting>` and their defaults:
+
+>>> data = maps.get_model_data('PyChop_fit')
+>>> data.defaults
+{'e_init': 500, 'chopper_frequency': 400}
+
+which can be used to do a quick initial test:
+
+>>> import numpy as np
+>>> model = maps.get_resolution_function('PyChop_fit', chopper_package='B')
+>>> energies = np.arange(0, data.defaults['e_init'], 0.5)
+>>> resolution = model(energies)
+
+by plotting the results (using a simple script such as):
+
+.. code-block:: Python
+
+    import matplotlib.pyplot as plt
+
+    fig, ax = plt.subplots(figsize=(10, 6))
+
+    ax.plot(energies, resolution)
+
+    ax.set_xlabel('Energy transfer (meV)', fontsize=20)
+    ax.set_ylabel('Sigma (meV)', fontsize=20)
+    ax.tick_params(labelsize=14)
+
+    plt.show()
+
+which will yield:
+
+.. image:: ./optimise_experiment_pictures/quick_example.png
+
+
+With this simple default plot, it should be possible to get an idea of the
+:term:`instrument` properties and check whether the other choices (e.g.
+:term:`configuration`) made seem about right. If everything checks out, the next
+step is to plot the resolution at different combinations of the
+:term:`settings<setting>`:
+
+>>> data.restrictions
+{'e_init': [0, 2000], 'chopper_frequency': [50, 601, 50]}
+
+The ``restrictions`` provide information about the allowed values for each
+:term:`setting`. In the case of the MAPS instrument, there are two different
+settings:
+
+* ``e_init`` for which all values between ``0`` and ``2000`` meV are allowed
+* ``chopper_frequency`` for which all values between ``50`` and ``600``
+  (inclusive) at an interval of ``50`` are allowed.
+
+However, depending on the instrument and the amount of scientific knowledge
+about the system, these values can be further constrained. For example, MAPS is
+a direct-geometry instrument, which means that only the values of energy
+transfer up to the value of ``e_init`` (the incident energy) are observable.
+Therefore, if it is known that in a given system features between 100 and 600
+meV are of interest, the ``e_init`` setting can be restricted further to values
+above 600 meV:
+
+>>> max_feature = 600
+>>> test_e_init = np.arange(max_feature, data.restrictions['e_init'][0], 100)
```

Comment:
The plot looks right, but I'm surprised that `data.restrictions['e_init'][0]` would be the _upper_ energy limit? Is that correct? Either way, perhaps `max(data.restrictions['e_init']` would be clearer.

Reply:
no, you're right, that is a mistake. `max` seems reasonable